packages/ring-client-api/ring-camera.ts

```diff
@@ -378,27 +377,16 @@ export class RingCamera extends Subscribed {
   }
 
   private async createStreamingConnection(options: StreamingConnectionOptions) {
-    if (this.isRingEdgeEnabled) {
-      const auth = await this.restClient.getCurrentAuth()
-      return new RingEdgeConnection(auth.access_token, this, options)
-    }
-
-    const liveCall = await this.restClient
-      .request<LiveCallResponse>({
+    const response = await this.restClient
+      .request<SocketTicketResponse>({
         method: 'POST',
-        url: this.doorbotUrl('live_call'),
+        url: 'https://app.ring.com/api/v1/clap/ticket/request/signalsocket',
       })
       .catch((e) => {
-        if (e.response?.statusCode === 403) {
-          const errorMessage = `Camera ${this.name} returned 403 when starting a live stream.  This usually indicates that live streaming is blocked by Modes settings.  Check your Ring app and verify that you are able to stream from this camera with the current Modes settings.`
-          logError(errorMessage)
-          throw new Error(errorMessage)
-        }
```

Comment:
Fun fact - I assumed we would still get these 403s for cameras that had live view disabled due to Modes. Looks like that's not the case! Just tried it and it let me stream from a camera which was supposed to be disabled. Ring app won't let me, but I could stream via ring-client-api 😂. Not sure if we should call this a bug or a feature, but thought I'd note it here 😄

Reply:
Yep, exactly why I removed this because, to my surprise, it just streamed anyway.  It didn't surprise me that a ticket would still be issued, but I was surprised the socket still accepted the livecall command.  I decided to just allow this in the ring-client-api PR since the Ring API technically allows it and leaves it up to the client side to enforce.

I'm not saying this is the right decision, just the one I made at the time.  I personally saw it as a feature, but I haven't decided whether to enforce or not in ring-mqtt itself.